### ISSUE TITLE
3.5.11 cannot be installed cleanly. upgrade works

### ISSUE THREAD

**Issue by dobbleg1000** [croneter/PlexKodiConnect#1702]
## Help yourself
* I did try to restart Kodi :-)
* I checked the [PKC Frequently Asked Questions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/faq)
* I did try to reset the Kodi database by going to `PKC Settings -> Advanced -> "Reset the database and optionally reset PlexKodiConnect"` and then hitting YES, NO
* I did check the [existing issues on Github](https://github.com/croneter/PlexKodiConnect/issues)

## Describe the bug
Try to install 3.5.11 and it gets could not sign into plex.tv Try again later. Installing 3.5.8 works

## To Reproduce
1. Uninstall PKC
2. Try to reinstall PKC beta

## Expected behavior
Provides code for plex.tv/link

## You need to attach a KODI LOG FILE!
A Kodi debug log file is needed that you recorded while you reproduced the bug. **Do clean your log of all Plex tokens (="Plex passwords")!!!** .
1. Activate Kodi's debug logging by going to the Kodi `Settings` -> `System` -> `Logging`. Then toggle the `Enable debug logging` setting.
2. Restart Kodi to start with a "fresh" log file.
3. Reproduce the bug.
4. Follow the [Kodi instructions](http://kodi.wiki/view/Log_file/Easy) to grab/share the Kodi log file. Usually only `kodi.log` is needed
    * You can [find the log file here](http://kodi.wiki/view/Log_file/Advanced#Location)
5. **Delete all references to any of your Plex tokens** by searching for `X-Plex-Token` and `accesstoken` and replacing the strings just after that!
    * It's easiest if you copy your token, then use Search&Replace for the entire log file
    * You don't want others to have access to your Plex installation....
6. Drop your log file here in this issue. Or use a free pasting-service like https://pastebin.com and include the link to it here
    
I am aware that I can delete Plex tokens that I accidentially posted by following the [instructions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug#i-published-my-plex-token-to-some-forum-or-github-anyone-can-now-access-my-plex-server)

**Comment by dobbleg1000** [comment]
[kodi.log](https://github.com/croneter/PlexKodiConnect/files/7508621/kodi.log)

**Comment by croneter** [comment]
Maybe the brand new beta version 3.5.12 also fixes this issue..?

**Comment by croneter** [comment]
Error here was
```
<general>: Traceback (most recent call last):
<general>:   File "/usr/lib/python3.9/multiprocessing/util.py", line 224, in __call__
<general>:     
<general>: res = self._callback(*self._args, **self._kwargs)
<general>: 
<general>:   File "/usr/lib/python3.9/multiprocessing/pool.py", line 692, in _terminate_pool
<general>:     
<general>: cls._help_stuff_finish(inqueue, task_handler, len(pool))
<general>: 
<general>:   File "/usr/lib/python3.9/multiprocessing/pool.py", line 947, in _help_stuff_finish
<general>:     
<general>: inqueue.get(block=False)
<general>: 
<general>: _queue
<general>: .
<general>: Empty
```
This is due to a f*-upped Python mutiprocessing implementation. Which PKC is NOT using anymore!!

**Comment by croneter** [comment]
Any news here?

**Comment by showkubii** [comment]
> Any news here?

This issue persists on PKC 3.5.13

[kodi.log](https://github.com/croneter/PlexKodiConnect/files/7555780/kodi.log)

Personally, I solved the problem by

`Settings>Advanced>Generate a new unique device Id>Reboot Kodi`

**Comment by croneter** [comment]
`Settings>Advanced>Generate a new unique device Id>Reboot Kodi`

That makes literally Zero sense. I'm thinking that it's a "hit or miss" whether PKC can be installed or not and you were simply lucky after doing that. 

@dobbleg1000 your status?

**Comment by croneter** [comment]
Thanks a ton for these bug reports, very valuable, @dobbleg1000 and @showkubii 

Issue should be fixed with version 3.5.14

**Comment by showkubii** [comment]
> Thanks a ton for these bug reports, very valuable, @dobbleg1000 and @showkubii
> 
> Issue should be fixed with version 3.5.14

Thanks for the quick fix.
